bin/ghe-backup

```diff
@@ -41,8 +41,16 @@ done
 # shellcheck source=share/github-backup-utils/ghe-backup-config
 . "$( dirname "${BASH_SOURCE[0]}" )/../share/github-backup-utils/ghe-backup-config"
 
+# Check that the parallel_command is pointing to moreutils parallel
+if ! $parallel_command -h | grep "parallel \[OPTIONS\] command -- arguments" &>/dev/null; then
```

Comment:
```suggestion
if ! $parallel_command -h | grep -q "parallel \[OPTIONS\] command -- arguments"; then
```

No need for redirection with `-q`.

I also suggest putting this into `share/github-backup-utils/ghe-backup-config` and combine it with where you're defining `$parallel_command` as you can't really have one without the other.

As it stands, this check would also need to be added to `ghe-restore` if you decide to implement parallel restores. Moving it now will make the `ghe-restore` get the functionality and check for free.

Reply:
I implemented `grep -q` as you suggested and moved the moreutils paralllel check into `ghe-backup-config` so it may be referenced later from `ghe-restore`.